version

```diff
@@ -0,0 +1 @@
+# Used by Concourse for semantic versioning/tagging your github-release
```

Comment:
I'd probably just initialise it to `0.1`. If someone is using this repo as a basis they're almost certainly starting from scratch

Reply:
No worries this has been updated